src/picongpu/include/particles/particleToGrid/ComputeGridValuePerFrame.def

```diff
@@ -93,7 +107,21 @@ struct CreateDensityOperation
     typedef FieldTmpOperation< ParticleDensity, T_Species > type;
 };
 
-/* ChargeDensity */
+/** Charge Density Operation for Particle to Grid Projections
+ *
+ * Derives a scalar charge density field from a particle species at runtime.
+ * Each value is mapped per cell according to the species' spatial shape.
+ *
+ * @note for species that do not change their charge state, this is identical
+ *       to the density times a constant for the charge,
```

Comment:
OPTIONAL:
better: `a constant for the charge` -> `the (constant) particle's charge`